### PR TITLE
Internal release

### PR DESCRIPTION
**Pull request by Dhan-shri**
Merging all code to Main

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -34,7 +34,7 @@ android {
         }
 
         getByName("debug") {
-            buildConfigField("String", "SERVER_URL", "\"https://dev-api.merakilearn.org/\"")
```
Comment (Dhan-shri) [inline app/build.gradle.kts:37]: just checked

File: app/build.gradle.kts [hunk 2]
```diff
@@ -34,7 +34,7 @@ android {
...
+            buildConfigField("String", "SERVER_URL", "\"https://merd-api.merakilearn.org/\"")
```
Comment (Dhan-shri) [inline app/build.gradle.kts:37]: hi

**Review by damanpreet97 — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/merakilearn/datasource/network/model/UploadCredentials.kt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    val ACL : String,
+
+    @Json(name = "Body")
+    val Body : String?,
+
+    @Json(name = "Credentials")
+    val Credentials : Credentials,
+
+    @Json(name = "project_id")
+    val project_id : String
```
[the remark this answers is not in the document]
Comment (Dhan-shri) [reply app/src/main/java/org/merakilearn/datasource/network/model/UploadCredentials.kt]: @damanpreet97 Done check once

File: app/src/main/java/org/merakilearn/datasource/network/UserUpdateName.kt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class UserUpdateName(
+
+    @Json(name = "name")
+    val name:String,
+
+    @Json (name = "profile_picture")
+    val profile_picture:String?
```
[the remark this answers is not in the document]
Comment (Dhan-shri) [reply app/src/main/java/org/merakilearn/datasource/network/UserUpdateName.kt]: @damanpreet97 Done check once. it is not using any column

File: app/src/main/java/org/merakilearn/FileChooserActivity.kt [hunk 5]
```diff
@@ -84,8 +84,21 @@ class FileChooserActivity : AppCompatActivity() {
                             onBackPressed()
                         }
                     }
+                    scheme.compareTo("https") == 0 -> {
```
[the remark this answers is not in the document]
Comment (Dhan-shri) [reply app/src/main/java/org/merakilearn/FileChooserActivity.kt]: Done check once

**Review by damanpreet97 — APPROVED** [review]

**Pull request merged**